### DISCUSSION TITLE
The word "safari" means what in Swahili?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#718] [Galaxy Trivia] [answered]
🦛🐆🦓🦒🐃

**Comment by peterbaccaro** [comment]
The Swahili word safari means "**journey**", originally from the Arabic noun سفر, safar, meaning "journey", "travel", "trip", or "tour".
Safari entered the English language at the end of the 1850s thanks to explorer Richard Francis Burton.

**Comment by SANKARAMDAS** [comment, marked as answer]
Originally the word safari meant «journey» in the Swahili language, which in turn comes from Arabic س (“safara”), which means travel. The verb travel in Swahili is “safiri“, the noun travel is “safari“. In that case, these words are used for any type of travel. 
It's entered the English language at the end of the 1850s thanks to explorer Richard Francis Burton.

**Comment by thuongtruong1009** [comment]
The word "safari" in Swahili means "journey" or "expedition." 🦛🐆🦓🦒🐃
